anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/docx/oxml/text/paragraph.py

```diff
@@ -16,7 +16,8 @@ class CT_P(BaseOxmlElement):
     r = ZeroOrMore('w:r')
     hyperlink = ZeroOrMore('w:hyperlink')
     sdt = ZeroOrMore('w:sdt')
-
+    m = ZeroOrMore('m:oMath')
```

Comment:
we have files which has w:oMath tag in it right, so are we handling that case any where?

Reply:
w:oMath is handled slightly differently within docx from m:oMath so currently it has not been tested

Reply:
inside w:oMath we have w:r but in m:oMath we have m:r other than that what other differences do they have?

Reply:
since it is registered by default in __init__, what would happen if we re-register the class under another class name?
(register_element_cls('w:oMath',      CT_OnOff)
Or can we just add this under paragraphs same as the m:oMath without breaking anything?

Reply:
if it is already registered are you getting that tag object ?
if not, try to register it how it is done for m:oMath